### PR TITLE
Storage acropolis

### PR DESCRIPTION
**Pull request by bwhm**

### CODE REVIEW

File: roles/storage-providers/README.md [hunk 1]
```diff
@@ -1,35 +1,354 @@
...
+$ systemctl stop caddy
+```
+
+## Install and Setup the Storage Node
+
+First, you need to clone the repo:
+```
+$ git clone https://github.com/Joystream/storage-node-joystream.git
+$ cd storage-node-joystream
+$ yarn && yarn install
```
Comment (mnaamani) [inline roles/storage-providers/README.md]: just `yarn`, its equivalent of `yarn install`

File: roles/storage-providers/README.md [hunk 2]
```diff
@@ -1,35 +1,354 @@
...
+#### Setup and configure the storage node
+
+**Make sure you're [Joystream full node](https://github.com/Joystream/substrate-node-joystream) is fully synced before you move to the next step(s)!**
+
+On the machine/VPS you want to run your storage node:
+
+```
+# If you are not already in that directory:
+$ cd storage-node-joystream
+$ (yarn run) colossus signup <5YourJoyMemberAddress.json>
```
Comment (mnaamani) [inline roles/storage-providers/README.md]: with colossus as an aliases, can't this just be `colossus signup ...` ?

File: roles/storage-providers/README.md [hunk 3]
```diff
@@ -1,35 +1,354 @@
...
+---
+# If you want to stop the storage node, either to edit the storage-node.service file or some other reason:
+$ systemctl stop storage-node
+```
 
 # Troubleshooting
 If you had any issues setting it up, you may find your answer here!
 
-## Install newest version of yarn and node without `sudo` on linux
+## Install yarn and node without on linux
```
Comment (mnaamani) [inline roles/storage-providers/README.md]: I would recommend also linking to this: https://github.com/nodesource/distributions/blob/master/README.md
It has worked well for me

Comment (bwhm) [reply roles/storage-providers/README.md]: Added this as an alternative!

File: roles/storage-providers/README.md [hunk 4]
```diff
@@ -1,35 +1,354 @@
...
 - [Table of contents](#table-of-contents)
 - [Overview](#overview)
 - [Instructions](#instructions)
+  - [Initial setup](#initial-setup)
+  - [Install ipfs](#install-ipfs)
+  - [Setup Hosting](#setup-hosting)
+  - [Install and Setup the Storage Node](#install-and-setup-the-storage-node)
 - [Troubleshooting](#troubleshooting)
-  - [Install newest version of yarn and node without `sudo` on linux](#install-newest-version-of-yarn-and-node-without-sudo-on-linux)
+  - [Install yarn and node without on linux](#install-yarn-and-node-without-on-linux)
```
[the remark this answers is not in the document]
Comment (bwhm) [reply roles/storage-providers/README.md]: Done.

File: roles/storage-providers/README.md [hunk 5]
```diff
@@ -1,35 +1,354 @@
...
+## Install ipfs
+The new storage node uses [ipfs](https://ipfs.io/) as backend.
+```
+$ wget https://dist.ipfs.io/go-ipfs/v0.4.21/go-ipfs_v0.4.21_linux-amd64.tar.gz
+$ tar -vxf go-ipfs_v0.4.21_linux-amd64.tar.gz
+$ cd go-ipfs
+$ ./ipfs init --profile server
+$ ./install.sh
+# start ipfs daemon:
+$ (screen) ipfs daemon
```
[the remark this answers is not in the document]
Comment (bwhm) [reply roles/storage-providers/README.md]: Removed.

File: roles/storage-providers/README.md [hunk 6]
```diff
@@ -1,35 +1,354 @@
...
+```
+If you see `Daemon is ready` at the end, you are good!
```
[the remark this answers is not in the document]
Comment (bwhm) [reply roles/storage-providers/README.md]: It's in there already!

File: roles/storage-providers/README.md [hunk 7]
```diff
@@ -1,35 +1,354 @@
...
+Restart=on-failure
+RestartSec=3
+StartLimitInterval=600
+
+[Install]
+WantedBy=multi-user.target
+```
+Save and exit. Close the `ipfs daemon` if it's still running, then:
+```
+$ systemctl start ipfs
```
[the remark this answers is not in the document]
Comment (bwhm) [reply roles/storage-providers/README.md]: Added (for all)

File: roles/storage-providers/README.md [hunk 8]
```diff
@@ -1,35 +1,354 @@
...
+```
+# Storage Node API
+https://<your.cool.url> {
+    proxy / localhost:3000 {
+        transparent
+    }
+    header / {
+        Access-Control-Allow-Origin  *
+        Access-Control-Allow-Methods "GET, PUT, HEAD, OPTIONS"
+        #Access-Control-Allow-Headers "content-type, range, bytes"
```
[the remark this answers is not in the document]
Comment (bwhm) [reply roles/storage-providers/README.md]: Removed!

File: roles/storage-providers/README.md (hunk 1 above)
Comment (bwhm) [reply roles/storage-providers/README.md]: Removed

File: roles/storage-providers/README.md (hunk 2 above)
Comment (bwhm) [reply roles/storage-providers/README.md]: Added a comment on that.

**Pull request merged**